oas_docs/output/kibana.serverless.yaml

```diff
@@ -203,6 +203,36 @@ tags:
       > If the API key that created a rule is deleted, or the user that created the rule becomes inactive, the rule will stop running.
 
       To create and run rules, the user must meet specific requirements for the Kibana space. Refer to the [Detections requirements](https://www.elastic.co/guide/en/security/current/detections-permissions-section.html) for a complete list of requirements.
+
+
+      ## Security Detections API Alerting API Key Behavior
+
+      The following table shows how Security Solution Detections APIs map to the underlying API key regeneration behavior:
+
+      | Security Solution API | Creates New API Key |
+      |----------------------|---------------------|
+      | Create Rule (enabled) (`POST /api/detection_engine/rules`) | Yes |
+      | Create Rule (disabled) (`POST /api/detection_engine/rules`) | No |
+      | Update Rule (`PUT /api/detection_engine/rules`) | Yes (+ See notes for Rule Enable) |
```

Comment:
For all the "update" operations (update, patch, bulk edit, import, upgrade prebuilt rules) it seems that an API key is not generated if there is no key on the rule yet. So in general updating a rule creates a new API key if there is an API key already (except for exception lists post-9.2), and enabling a rule creates a new API key if there **isn't** an API key already. Other operations should have no effect on the API key (except delete, which deletes the key with the rule).